### PR TITLE
List display

### PR DESCRIPTION
**Pull request by alicelouise95**
Changed the entrypoint to a UINavigationController, added placeholder cell tap nav

### CODE REVIEW

File: lia-app-uikit/ListTableViewController.swift [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    
+    // Creating the navigation on cell tap.
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        navigateToDetail(forItemAt: indexPath.row)
+    }
+    
+    // Placeholder for the navigation function
+    func navigateToDetail(forItemAt index: Int) {
+        print("Placeholder for navigation. Item tapped: \(items[index])")
+
```
Comment (henrikssonkevin) [inline lia-app-uikit/ListTableViewController.swift]: Remove space

File: lia-app-uikit/ListTableViewController.swift [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    override func tableView(_ tableView: UITableView, canEditRowAt indexPath: IndexPath) -> Bool {
+        return true
+    }
+    
+    // Creating the functionality to edit the table view
+    override func tableView(_ tableView: UITableView, commit editingStyle: UITableViewCell.EditingStyle, forRowAt indexPath: IndexPath) {
+        if editingStyle == .delete {
+            items.remove(at: indexPath.row)
+            tableView.deleteRows(at: [indexPath], with: .fade
+            )
```
Comment (henrikssonkevin) [inline lia-app-uikit/ListTableViewController.swift]: Move bracket one line up

File: lia-app-uikit/ListTableViewController.swift [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    
+    // Creating and configuring a cell for each row in the table.  
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "cell", for: indexPath)
+        let myImage = UIImage(systemName: "star.fill")
+        cell.imageView?.image = myImage
+        cell.textLabel?.text = "Placeholder title"
+        cell.accessoryType = .disclosureIndicator
+        return cell
+
```
Comment (henrikssonkevin) [inline lia-app-uikit/ListTableViewController.swift]: Remove space

File: lia-app-uikit/ListTableViewController.swift [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    }
+    
+    // Creating and configuring a cell for each row in the table.  
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "cell", for: indexPath)
+        let myImage = UIImage(systemName: "star.fill")
+        cell.imageView?.image = myImage
+        cell.textLabel?.text = "Placeholder title"
+        cell.accessoryType = .disclosureIndicator
+        return cell
```
Comment (henrikssonkevin) [inline lia-app-uikit/ListTableViewController.swift:45]: Nice but this does not count as a custom cell. This is basically using the standard UITableViewCell provided by apple which does have some customizability. This is fine for now but I will create a new task for making a custom cell class.

**Review by henrikssonkevin — APPROVED** [review]
Looking good! A few minor stuff. You will probably run into some issues regarding `IPHONEOS_DEPLOYMENT_TARGET = 16.6;` so see if you are able to revert that change as I did with filip this morning. Let me know if help is needed.

File: lia-app-uikit/ListTableViewController.swift (hunk 4 above)
Comment (alicelouise95) [reply lia-app-uikit/ListTableViewController.swift:45]: Yeah I think I was a bit confused by what you meant by a custom cell. Sounds good tho

**Pull request merged**